platform_versioned_docs/version-24.1/cloud/changelog.mdx

```diff
@@ -5,41 +5,38 @@ date: "4 July 2024"
 tags: [changelog]
 ---
 
-### v24.2.0_cycle23 Cloud - 2 September 2024
-
-- Fix: [Data Explorer] enable task working directory navigation using Data Explorer in personal workspaces by adding routing
-- Fix: Remove -/+ increment buttons from numeric input components in Platform, including compute environment and launch form interfaces
-- Fix: Hide compute environment variables section actions when component is disabled (previously indicated that environment variables could be edited in this state)
-
-### 24.2.0_cycle21 Cloud - 13 August 2024
-
-- Feature: Add global Nextflow configuration support in compute environments
-- Feature: Add flexibility for pipeline names in workspaces
-- Feature: Add tag propagation to launch templates
-- Feature: Add managed identities support for manual Azure Batch compute environments
-- Feature: Implement custom launch container logic
-- Fix: Improve workflow launch screen look and feel
-- Fix: Allow pipeline work directory to be changed during pipeline launch
-- Fix: Handle special characters in prompt modal confirmation text regex
+### 24.2.0 - 2 September 2024
+
+- Data Explorer: Enable task working directory navigation using Data Explorer in personal workspaces by adding routing
+- Remove -/+ increment buttons from numeric input components in Platform, including compute environment and launch form interfaces
+- Hide compute environment variables section actions when component is disabled (previously indicated that environment variables could be edited in this state)
+
+### 24.2.0 - 13 August 2024
+
+- Add global Nextflow configuration support in compute environments
+- Improved flexibility for pipeline names in workspaces
```

Comment:
If we change the format that we documented [here](https://seqera.atlassian.net/wiki/spaces/EDU/pages/102465547/Release+notes+and+changelogs#Changelog-format), then we'll have to change the same for all versions of Enterprise. So this PR creates a disparity between the formats of Cloud and Enterprise changelogs.

Reply:
I noted that adding those prefixes didn't apply to everything so it looked disjointed - and it's not easy to remember the order off-hand which means it makes updating the docs tiresome. 

If we don't add those prefixes, we have to write more descriptive sentences to make the changelog notes more meaningful which I believe is a good approach. It also rolls into another change I'd like to implement which is to separate bugs, features, fixes into their own sections (that'll be the next PR). 

I don't necessarily agree that we have to change the format for everything - it can be a change going forward. It might make more sense/be more viable in the next PR because it'll be a more obvious break from the previous format.

Reply:
I'm open to including the changes I have in mind for the next PR into this PR so we can see how it looks.